core/common/src/ipc/BackendIpc.ts

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module IpcSocket
+ */
+
+import { IModelError, IModelStatus } from "../IModelError";
+import { IpcInterface, IpcInvokeReturn, IpcSocketBackend, iTwinChannel, RemoveFunction } from "./IpcSocket";
+
+/**
+ * This class provides backend support for Ipc operations. It must be initialized with a platform-specific
+ * implementation of the [IpcSocketBackend]($common) interface at startup, before calling [IModelHost.startup]($backend).
+ * @internal
+ */
+export class BackendIpc {
+  private static _ipc: IpcSocketBackend | undefined;
+  /** Get the implementation of the [IpcSocketBackend]($common) interface. */
+  public static get ipc(): IpcSocketBackend { return this._ipc!; }
+  /**
+   * initialize backend support for Ipc
+   * @param ipc The platform-specific implementation of the [IpcSocketBackend]($common) interface
+   */
+  public static initialize(ipc: IpcSocketBackend) { this._ipc = ipc; }
+  /** Determine whether Ipc is available for this backend. This will only be true if [[initialize]] has been called on this class. */
+  public static get isValid(): boolean { return undefined !== this._ipc; }
+
+  /**
+   * Send a message to the frontend over an Ipc channel.
+   * @param channel the name of the channel matching the name registered with [FrontendIpc.handleMessage]($frontend).
+   * @param data The content of the message.
+   */
+  public static sendMessage(channel: string, ...data: any[]) {
+    return this._ipc!.send(iTwinChannel(channel), ...data);
+  }
+}
+
+/**
+ * Base class for all implementations of [IpcInterface]($frontend).

```

Comment:
`IpcInterface` is defined in common, not frontend, no?

Reply:
yup, I moved it. Will fix.

Reply:
fixed